### ISSUE TITLE
Runtime Metrics no longer working

### ISSUE THREAD

**Issue by MartinLoeper** [DataDog/dd-trace-js#1029]
**Describe the bug**
We updated our dd-trace-js version from 0.22.1 to 0.23.0 and immediately after the new deployment the runtime metrics stopped from appearing in the DataDog UI.

Did something change with regards to runtime metrics configuration?
We enable them inside the dd-trace init part as follows:

```ts
tracer.init({
    enabled: true,
    service: "core",
    logInjection: true,
    env: process.env.NODE_ENV,
    runtimeMetrics: true,
    analytics: true,
});
```

**Environment**

* **Operation system:** Amazon Linux 2
* **Node version:** node v12.18.2
* **Tracer version:** 0.23.0
* **Agent version:** 7.21.0

**Comment by rochdev** [comment]
There was an update to allow configuring a different hostname for DogStatsD specifically, but it should still fallback to the trace agent hostname when not set. There were also a few changes related to service/env/version tags but again it shouldn't have affected runtime metrics.

Are you missing all metrics or only some of them? Also, if you look into the Metric Explorer can you find the metrics maybe with the wrong tags or are they completely missing from the entire account?

**Comment by MartinLoeper** [comment]
It looks like no runtime metrics are reported at all from the new fleet of containers running dd-trace-js v0.23.0. I will suggest the team to downgrade the dd-trace-js version for the application in order to verify that it is indeed related to this specific package update and no other side effect exists.

**Comment by MartinLoeper** [comment]
I can confirm that downgrading the dd-trace-js back to v0.22.1 brings back the runtime metrics.

**Comment by rochdev** [comment]
Fix released in 0.23.2
